### PR TITLE
[#36] Inspect code style

### LINKED ISSUE #36: Inspect code style
Some codes do not follow the code style guide rule.
Find the mistake and patch it.

Here is the code style rule in korean.
https://github.com/ztkmkoo/dss/wiki/Code-Style(KR)

### PR DESCRIPTION
**Pull request by sia-txt**
Fixes #36 

Continuous inspecting is needed.

### CODE REVIEW

**Comment by Alencion** [comment]
Hi, I found the unused import statement in dss-server/test/DssRestServerTest.java. 

Could you remove it?

**Comment by Amazansia** [comment]
> Hi, I found the unused import statements in dss-server/test/DssRestServerTest.java.
> 
> Could you remove it?

Hi! Thank you for reviewing. I think I missed checking all the import statements...I'll check and revise them.

**Review by vincentj2 — APPROVED** [review]

**Comment by Amazansia** [comment]
I learned a lot after 5 commits...and typing "git commit -m [#36] Inspect code style" 5 times. 

There are **some changes** in final commit: **to reviewers** _(Thanks in advance!)_
1. Plz check dss-server.test.DssRestServerTest.java. According to code style rules, every static thing has to be placed at first. 
But here's a thing, that static things are too many. So I think I did it correctly, but still confusing...
2. I removed unused import statements at DssRestServerTest.java. And there was no such unused import statements in other files.
3. I edited pom.xml in dss-server, to catch right version of external library. 
like...
```
<properties>
        <!-- Dependencies version -->
        **<dss-core.version>0.4.4</dss-core.version>**
        <akka.version>2.6.3</akka.version>
</properties>
```

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestServiceActor.java [hunk 1]
```diff
@@ -65,7 +66,7 @@ private DssRestServiceActor(ActorContext<DssRestServiceActorCommand> context, Ds
         return Behaviors.same();
     }
 
-    private void replyRequest(DssRestServiceActorCommandRequest request, HttpResponseStatus status, DssRestServiceResponse response) {
+    private void replyRequest (DssRestServiceActorCommandRequest request, HttpResponseStatus status, DssRestServiceResponse response){
```
Comment (Alencion) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestServiceActor.java:69]: this line too, 

and add space between parameter and statement.

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestServiceActor.java [hunk 2]
```diff
@@ -79,7 +80,7 @@ private void replyRequest(DssRestServiceActorCommandRequest request, HttpRespons
                 );
     }
 
-    private void replyRequest(DssRestServiceActorCommandRequest request, HttpResponseStatus status) {
+    private void replyRequest (DssRestServiceActorCommandRequest request, HttpResponseStatus status){
```
Comment (Alencion) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestServiceActor.java:83]: this line too.

**Review by ParkSale — CHANGES_REQUESTED** [review]

File: dss-core/src/test/java/io/github/ztkmkoo/dss/core/network/rest/handler/DssRestChannelInitializerTest.java [hunk 3]
```diff
@@ -29,6 +29,11 @@
 
     private static final ActorTestKit testKit = ActorTestKit.create();
 
+    @AfterAll
+    public static void cleanUp() {
+        testKit.shutdownTestKit();
+    }
```
[the remark this answers is not in the document]
Comment (Amazansia) [reply dss-core/src/test/java/io/github/ztkmkoo/dss/core/network/rest/handler/DssRestChannelInitializerTest.java:35]: I agree with @Doyuni , maybe we should discuss about some exceptions of code style rules like this case.

File: dss-server/pom.xml [hunk 4]
```diff
@@ -35,12 +35,13 @@
...
 	<properties>
         <!-- Dependencies version -->
+        <dss-core.version>0.4.4</dss-core.version>
         <akka.version>2.6.3</akka.version>
     </properties>
     <dependencies>
         <!--dss core model-->
         <dependency>
-            <groupId>${project.groupId}</groupId>
+            <groupId>io.github.ztkmkoo</groupId>
```
[the remark this answers is not in the document]
Comment (RulLu16) [reply dss-server/pom.xml:44]: It's still not changed. Check it please! :)

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssLogLevel.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    ;
+
+    private LogLevel level;
+
+    DssLogLevel( LogLevel level) {
+        this.level = level;
+    }
+
+    public LogLevel getNettyLogLevel() {
+        return level;
```
Comment (RulLu16) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssLogLevel.java:20]: These codes are #112 commit. So I think you need to rebase commits! Check please :)

Comment (Amazansia) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssLogLevel.java:20]: Oh I don't know why git consider these codes as new codes after I pulled correctly, I think... Okay, I'll revise this! Thank you for reviewing :>

**Review by ztkmkoo — APPROVED** [review]
@Amazansia 
Wow, You fix all of the reviews. 
Good job

**Review by Alencion — APPROVED** [review]

**Review by RulLu16 — APPROVED** [review]

**Comment by Amazansia** [comment]
**God bless this commit and let it be closed, Amen.** (I'm a atheist)

**Pull request merged**